bundles/org.openhab.voice.pollytts/src/main/java/org/openhab/voice/pollytts/internal/PollyTTSService.java

```diff
@@ -167,17 +160,15 @@ public AudioStream synthesize(String inText, Voice voice, AudioFormat requestedF
         // now create the input stream for given text, locale, format. There is
         // only a default voice
         try {
-            File cacheAudioFile = pollyTTSImpl.getTextToSpeechAsFile(text, voice.getLabel(),
+            InputStream pollyAudioStream = pollyTTSImpl.getTextToSpeech(text, voice.getLabel(),
                     getApiAudioFormat(requestedFormat));
-            if (cacheAudioFile == null) {
+            if (pollyAudioStream == null) {
                 throw new TTSException("Could not read from PollyTTS service");
             }
             logger.debug("Audio Stream for '{}' in format {}", text, requestedFormat);
-            AudioStream audioStream = new PollyTTSAudioStream(cacheAudioFile, requestedFormat);
+            AudioStream audioStream = new PollyTTSAudioStream(pollyAudioStream, requestedFormat);
             return audioStream;
-        } catch (AudioException ex) {
-            throw new TTSException("Could not create AudioStream: " + ex.getMessage(), ex);
-        } catch (IOException ex) {
+        } catch (AmazonPollyException ex) {
```

Comment:
The javadoc of `getTextToSpeech` is mentioning `IOException` but not `AmazonPollyException`.

Reply:
@dalgwen : any feedback?

Reply:
OK, I fixed the javadoc.
The IOException is no longer possible as it was thrown by file related operation used by the custom cache.
The AmazonPollyException was not catched before (it is a RuntimeException) and I thought it was a good thing to catch it here.

But caution : I did not tested it. I still have the same resolution issue.